src/context/WalletProvider/NativeWallet/components/LegacyLogin.tsx

```diff
@@ -28,19 +36,27 @@ export const LegacyLogin = () => {
   const {
     handleSubmit,
     register,
-    reset,
     formState: { errors, isSubmitting },
   } = useForm({ shouldUnregister: true })
 
   const translate = useTranslate()
 
+  const isLoginError = (err: any): err is LoginError => err && typeof err.message === 'string'
+
   const onSubmit = async (values: FieldValues) => {
-    // @TODO(NeOMakinG): Remove this when we handle the error response from the API
-    if (true) {
+    try {
+      await new Promise((resolve, reject) => resolve(null))
+      // TODO: use response to create wallet vault.
```

Comment:
We need to add `validateMnemonic` to verify that the decryption returned a valid mnemonic

Reply:
I added the condition but atm it does nothing interesting, I would need more information on what should we do in case the mnemonic is invalid